test/test_view/test_bloomberg.py

```diff
@@ -77,10 +78,10 @@ def test_login_create_account_fail(self, mock_one_login, mock_create_account):
         mock_auth.is_authenticated = False
         mock_one_login.return_value = mock_auth
         mock_auth.get_attributes.return_value = {'UUID': [u'1234567'], 'FirstName': [u'test'], 'emailAddress': ['test@test.com'], 'LastName': [u'test'], 'PVFLevels': [u'PVF_GUTS_3'], 'LoginID': [u'test']}
-        mock_create_account.side_effect = Exception()
```

Comment:
Is this line still needed in order to trigger the exception?

Reply:
It is passing the test without that line, so I would say no.

Reply:
```
with assert_raises(Exception) as context:
            res = self.app.post('/bloomberg/login', method='POST', content_type='multipart/form-data', data={'RelayState': redirect_url})
            assert_true('Auto-account creation error' in context.exception)
            assert res.status_code == 302, res.status_code
```

Reply:
Yes, I think this should be working correctly now. Thanks!